Xamarin.Forms.Platform.GTK/VisualElementRenderer.cs

```diff
@@ -268,6 +227,21 @@ protected virtual void UpdateNativeControl()
             UpdateSensitive();
         }
 
+        protected bool IsAnimationRunning(VisualElement element)
```

Comment:
We will need to refactor this part. Now its working but no without problems. Will do in another branch / PR.